cookbook/specs/spec_add_namada.json

```diff
@@ -0,0 +1,651 @@
+{
+  "deposit": "10000000ulava",
+  "proposal": {
+    "title": "Add spec: NAMADA SE testnet",
+    "description": "Adding new specification support for relaying data on NAMADA",
+    "specs": [
+      {
+        "index": "NAMTSE",
+        "name": "namada se testnet",
+        "enabled": true,
+        "reliability_threshold": 268435455,
+        "data_reliability_enabled": false,
+        "block_distance_for_finalized_data": 0,
+        "blocks_in_finalization_proof": 1,
+        "average_block_time": 6500,
+        "allowed_block_lag_for_qos_sync": 2,
+        "shares": 1,
+        "min_stake_provider": {
+          "denom": "ulava",
+          "amount": "47500000000"
+        },
+        "api_collections": [
+          {
+            "enabled": true,
+            "collection_data": {
+              "api_interface": "tendermintrpc",
+              "internal_path": "",
+              "type": "",
+              "add_on": ""
+            },
+            "apis": [
+              {
+                "name": "dump_consensus_state",
```

Comment:
if it's a tendermint chain, we are inheriting apis, rather than rewrite them for maintainability and readability
if you want to inherit only a part of the api collections you just overwrite the other imports to enabled:false by specifying the apiCollection with the same collection data
are there no other apis?

Reply:
Thx for the response! I didn't know that we could inherit only one API. I will prepare fixes 

> Are there no other APIs?

Namada is based on commetbft, but the application layer doesn't use cosmosSDK, the app layer is written from scratch in Rust. Only RPC available

Reply:
Maybe for this purpose, it would be great to have inherited spec only for terndemint/commetBFT API? 
https://docs.tendermint.com/v0.34/rpc/

Reply:
Yes if chains start using tendermint separately to cosmos definitely
Need to create a base tendermint and cosmos to
Inherit from it

Reply:
Refoctered spec with inheritance but got the error in GitHub actions, but `test_spec_full.sh` works well on my side

Reply:
the sdk has a statistical unitest error we are aware of